pyproject.toml

```diff
@@ -21,6 +21,7 @@ flask_testing = "^0.7.1"
 pytest = "^4.4"
 django = "^2.2"
 Flask = "^1.0.1"
+vcrpy = "^2.0"
```

Comment:
`vcrpy` looks interesting. Did you consider writing and committing a script to regenerate the fixtures? I think that would be valuable - I assume this is the sort of thing that `vcr` makes super easy.

Reply:
Yup, basically `rm -rf fixtures && poetry run pytest` to regenerate everything

Reply:
Ill add it to the readme instead :)